tests/Propel/Tests/Generator/Builder/Om/TableMapBuilderTest.php

```diff
@@ -378,11 +379,12 @@ public function testNormalizedColumnMapHasOnlyUniqueKeys()
         $schema = $reader->parseString($databaseXml);
         $table = $schema->getDatabase()->getTable('email');
 
-        $tableMapBuilder = new class($table) extends TableMapBuilder {
+        $tableMapBuilder = new class ($table) extends TableMapBuilder {
```

Comment:
PSR seems to not say anything about the style of an anonymous class declaration, hence I would follow the examples from the PHP manual. And those do not have spaces after `class`.

Reply:
There should be a code sniffer for for one or the other.

Reply:
The code sniffer/fixer actually adds the space after class. I don't like it, but fighting the formatter is pointless.